packages/button/Button.test.tsx

```diff
@@ -48,7 +49,7 @@ describe( Button.is, () => {
       it( `should handle click`, () => {
 
         let clickTriggered = false;
-        const handleClick = ( e: MouseEvent ) => { clickTriggered = true; };
+        const handleClick = ( e: ClickEvent ) => { clickTriggered = true; };
```

Comment:
this is rather wrong, pls see first review commit. `ClickEvent` is function in this case, so just remove it and use appropriate native event type ( in this case `MouseClickEvent` )